bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateAsterismsTreeTableModel.java

```diff
@@ -95,9 +96,11 @@ public Object getValue(Row row) {
         // (XXX There might be another way to get the parent node)
         private Row _parent;
 
+        private final Option<Long> _when;
+
         // A row corresponding to a GemsGuideStars object
         // (top level node, displays the Strehl info and checkbox)
-        Row(GemsGuideStars gemsGuideStars, List<Row> children) {
+        Row(GemsGuideStars gemsGuideStars, List<Row> children, Option<Long> when) {
```

Comment:
What this is saying is let's display the asterism with ra/dec at the time set on the scheduling block right?
Shouldn't this be done when AGS calculates the targets instead and then ra/dec of each target would already be corrected? I may be miss-understanding this though